client/pages/sections/protocols/protocols.js

```diff
@@ -83,6 +80,16 @@ class Protocol extends PureComponent {
     const protocolState = this.getProtocolState();
     const isActive = this.isActive(protocolState);
 
+    const conditionalFateOfAnimalFields = renderFieldsInProtocol(this.props.project['fate-of-animals']);
+
+    // Ensure options array exists, and is initialised properly
+    if (!this.props.sections.fate.fields[0].options) {
+      this.props.sections.fate.fields[0].options = [];
+    }
+
+    // Update the options array with unique fields
+    this.props.sections.fate.fields[0].options = conditionalFateOfAnimalFields;
```

Comment:
I don't know enough about this PR so just asking, is it okay to update props this way? Is everything behaving as expected?

If you check field.js file in this PR for example (probably not best example). Props is updated using state.

Reply:
renderField is based on props which are a list of fields; these fields have their state which is not affected by the above change as it only modifies the props in the parent component.